src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/EnumDataContract.cs

```diff
@@ -11,24 +11,19 @@
 using System.Xml;
 using System.Security;
 using System.Linq;
-
+using System.Diagnostics.CodeAnalysis;
 
 namespace System.Runtime.Serialization
 {
     internal sealed class EnumDataContract : DataContract
     {
         private readonly EnumDataContractCriticalHelper _helper;
 
-        public EnumDataContract() : base(new EnumDataContractCriticalHelper())
```

Comment:
are we sure these are not called by reflection?

Reply:
I've searched everywhere I can find, and I don't see anyone creating these through reflection. Also the tests are passing, so I think we are safe removing these dead constructors on internal classes.